spec/factories/notes.rb

```diff
@@ -0,0 +1,8 @@
+FactoryGirl.define do
+  factory :note do
+    notable { create(:person) }
```

Comment:
Okay.  I removed the notable from the notes factory.  But this leaves the default factory invalid and prevents being able to automatically test the translatable concern.  Is there a different pattern that I should be following here?